engine/src/cubos/engine/imgui/plugin.cpp

```diff
@@ -1,17 +1,18 @@
-#include <cubos/core/ui/imgui.hpp>
-
 #include <cubos/engine/imgui/plugin.hpp>
 #include <cubos/engine/window/plugin.hpp>
 
+#include "imgui.hpp"
```

Comment:
Use full include path for consistency.

Reply:
not quite, broad_phase.cpp also includes without full path. As these are next to each other I would prefer relative paths